requirements/constraints.txt

```diff
@@ -80,8 +76,8 @@ python-slugify<5.0.0
 # greater version has breaking changes and requires some migration steps.
 django-webpack-loader==0.7.0
 
-# celery requires click<8.0.0 which would be fixed once https://github.com/celery/celery/issues/6753 is done.
-click<8.0.0
+# required for celery>=5.2.0;<5.3.0
+click>=8.0,<9.0
```

Comment:
Could you move this after celery since that ordering is better alphabetically and because these are related?